src/_guides/language/language-tour.md

```diff
@@ -160,22 +166,23 @@ The following table lists the words that the Dart language treats specially.
 | deferred{{bii}}   | implements{{bii}}
 {:.table .table-striped .nowrap}
 
-<sup>1</sup> Words with the superscript **1**
-are **built-in identifiers**. Avoid using
-built-in identifiers as identifiers.
-A compile-time error happens if you try to
-use a built-in identifier for a class or type name.
-
-<sup>2</sup> Words with the superscript **2**
-are newer, limited reserved words related to **asynchrony** support
-added after Dart's 1.0 release.
-You can't use `async`, `await`, or `yield` as
-an identifier in any function body marked with `async`, `async*`, or `sync*`.
-For more information, see
-[Asynchrony support](#asynchrony-support).
+When necessary, the words marked with superscripts can sometimes be used as
+identifiers:
```

Comment:
Not sure that `sync*`, `async*` and `yield*` fits in the table above since they are not identifiers.

Even the plain `sync` or `async` are not reserved in any way.
They are no more reserved than `on`, `show` or `hide`, which are plain contextual keywords (means something only when they occur in a specific place, but are completely valid identifiers in every way otherwise).

Reply:
They're not reserved, but don't you think it'd be confusing to see them used as identifiers?

Reply:
Maybe I should have `async`, `on`, `show`, and `hide` in the table in a new category (contextual keywords). (Are there any others?)

This table is all about which words are are meaningful to Dart. I think any meaningful word might be confusing if used as an identifier, even if it's allowed, but I can soften the "avoid" language if you think that's important.

I'll remove `sync*`, `async*`, and `yield*`.